docs-website/sidebars.js

```diff
@@ -273,6 +273,28 @@ module.exports = {
         ],
       },
     ],
+    "Authorization & Authentication": [
+      {
+        type: "doc",
+        id: "docs/auth/README",
+        label: "Introduction",
```

Comment:
Let's order it as 

- Introduction
- Concepts
- Personal Access Tokens
- Frontend Authentication (Subpages can include the existing JAAS, OIDC pages, and even Aditya's new inviting users doc) 
- Metadata Service Authentication (Don't say GMS)